app/Console/Commands/UpdateSignupAndOrPostField.php

```diff
@@ -15,7 +15,7 @@ class UpdateSignupAndOrPostField extends Command
      *
      * @var string
      */
-    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts}';
+    protected $signature = 'rogue:updatefield {target} {targetOldValue} {targetNewValue} {--signups} {--posts} {--logfreq=1}';
```

Comment:
@chloealee I think the default should be something higher so that the logs don't get overloaded if it gets run without this option set. Maybe 10000?

Reply:
ah i was planning on having that as the default but then when actually running it doing something like `10000` but making that the default makes sense to me - thanks @katiecrane !